docs/tutorial/tutorial_recap.py

```diff
@@ -0,0 +1,34 @@
+import globus_sdk
+
+# this is the tutorial client ID
+# replace this string with your ID for production use
+CLIENT_ID = "61338d24-54d5-408f-a10d-66c06b59f6d2"
+
+# create your app
+my_app = globus_sdk.UserApp("my-user-app", client_id=CLIENT_ID)
+
+# create a client with your app
+groups_client = globus_sdk.GroupsClient(app=my_app)
+
+# print in CSV format
+print("ID,Name,Type,Session Enforcement,Roles")
+for group in groups_client.get_my_groups():
+    # parse the group to get data for output
+    if group.get("enforce_session"):
+        session_enforcement = "strict"
+    else:
+        session_enforcement = "not strict"
+    roles = "|".join({m["role"] for m in group["my_memberships"]})
+
+    print(
+        ",".join(
+            [
+                group["id"],
+                # note that 'name' could actually have commas in it, so quote it
+                f'"{group["name"]}"',
+                group["group_type"],
+                session_enforcement,
+                roles,
+            ]
+        )
+    )
```

Comment:
My gut reaction is that this this processing bit seems like it takes up too much screen space in an otherwise very small script when it's not exactly what the script is trying to teach. 

I'm not sure what suggestions I have though for other use cases to more simply demonstrate "using a response" though. I'll think about it some more but if I can't come up with a better alternative can be okay with this.

Reply:
I managed to squish it down to 4 lines total of printing/looping, which I think is good enough that the tutorial doc reads much better. I'm calling that good, but we can reopen this topic as needed.